projects/canopy/src/lib/page/docs/page.stories.ts

```diff
@@ -33,41 +68,142 @@ const createArgs = () => ({
 });
 
 const header = `
-  <header lg-header
-    [logo]="logo"
-    [logoAlt]="logoAlt"
-    [logoHref]="logoHref">
+  <header lg-header>
+    <lg-header-logo [src]="logo" [alt]="logoAlt" [href]="logoHref"></lg-header-logo>
   </header>
 `;
 
 const footer = `
-  <footer lg-footer
-    [logo]="logo"
-    [logoAlt]="logoAlt"
-    [copyright]="copyright"
-    [primaryLinks]="primaryLinks"
-    [secondaryLinks]="secondaryLinks">
-  </footer>
+<footer lg-footer>
```

Comment:
this block has the wrong indentation and needs to be indented 1 more step

Reply:
fixed